components/email/src/Email.svelte

```diff
@@ -737,6 +756,7 @@
   @import "../../theming/reset.scss";
   @import "../../theming/animation.scss";
   @import "../../theming/variables.scss";
+  @import "./styles/email.scss";
 
   $hover-outline-width: 2px;
```

Comment:
Should we include all the styles in here in `./styles/email.scss` while we're at it?

Reply:
Email styles are super soupy, I think it's more valuable to break it out piece by piece - but yeah we can put it all in there if we want.